assets/less/variables.less

```diff
@@ -46,5 +49,5 @@
 @skill-master-color: @theme-green;
 
 // Label styles
+@theme-label-border-color: darken(@theme-label-bg, 20%);
```

Comment:
I think this line should come after
```less
@theme-label-bg: lighten(@theme-blue, 15%);
```
else we might get variable undefined error